app/routes.php

```diff
@@ -1,15 +1,14 @@
 <?php
 declare(strict_types=1);
 
+use Psr\Http\Message\RequestInterface;
+use Slim\Http\Response;
 use Slim\App;
 use Slim\Interfaces\RouteCollectorProxyInterface as Group;
 
 return function (App $app) {
     $container = $app->getContainer();
 
-    $app->get('/', function ($request, $response, $args) use ($container) {
-        $renderer = $container->get('renderer');
-        return $renderer->render($response, "index.php", $args);
-    });
+    $app->get('/', \App\Controllers\ExampleController::class);
 
```

Comment:
correct me if i'm wrong but it seems that we're now losing the controller example that uses a renderer to render a page? i think it would be useful to have at least 1 route that displays a page?

or are we suggesting to have that as part of the teaching and they learn it that way?

Reply:
I think leave the previous route in there, that's rendering a page, the have the new json route under `/json` or something

Reply:
Yeah totally not suggesting that we remove the old route from this - I just knocked up a quick example to show how the autowiring and response decorator works